kubehub/proxmox/get_less_busy_node.py

```diff
@@ -0,0 +1,18 @@
+from proxmoxer import ProxmoxAPI
+
+
+def get_less_busy_node(host, password):
```

Comment:
why not pass cloud_provider_instance object instead host and password? why username hardcoded?

Reply:
- i already passing cloud provider instance objects, host and password is variable names
- need more explanation about username, by that you mean user_id in vm_group_add function or  user='root@pam' in ProxmoxAPI?